src/Elastic.Apm/Config/ConfigConsts.cs

```diff
@@ -17,6 +17,7 @@ public static class EnvVarNames
 		public static class DefaultValues
 		{
 			public const double TransactionSampleRate = 1.0;
+			public const string ServiceName = "unknown";
```

Comment:
nit: can we maybe call it "UnknownServiceName". Because this is kind of the fallback default value.. the real default value is what we discover.

Reply:
Done.